apps/bond/components/BondingProducts/Bonding/BondingList/useBondingList.jsx

```diff
@@ -216,15 +252,10 @@ const useAddCurrentLpPriceToProducts = () => {
             };
           } else {
             let currentLpPrice = null;
-            // NOTE: It could be uniswap for other chains hence this if case.
-            // (commented for now)
-            // if (dex === DEX.UNISWAP) {
-            //   currentLpPricePromise = contract.methods
-            //     .getCurrentPriceUniswap(originAddress)
-            //     .call();
-            //   currentLpPricePromiseList.push(currentLpPricePromise);
-            // } else
-            if (dex === DEX.BALANCER) {
+            if (dex === DEX.UNISWAP) {
+              currentLpPrice = getCurrentPriceUniswap(productList[i].token);
+              otherRequests[i] = currentLpPrice;
+            } else if (dex === DEX.BALANCER) {
               currentLpPrice = getCurrentPriceBalancer(productList[i].token);
               otherRequests[i] = currentLpPrice;
             } else if (dex === DEX.SOLANA) {
```

Comment:
we should refactor the `DEX.###` keys at a later point..

using `ABI.UNISWAP_V2` for example, same for `.SOLANA`, would be better to identify the project and the type of contract we're interacting with